internal/security/secretstore/certs_test.go

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
```

Comment:
author seems incorrect here given the git history, but if this is originally from the files that Tingyu wrote, I would just add yourself as another author here. just seems wrong that if I look in git history at who initially wrote this file I see you, but it says only Tingyu is the author